addons/true-youtube-links/userscript.js

```diff
@@ -9,6 +16,10 @@
         },
       }
     );
+    if (addon.self.disabled) {
+      continue;
+    }
```

Comment:
Technically you shouldn't skip the link here, you should add it to a list and handle it later if the addon gets reenabled. Which makes this code extra ugly.
dynamicDisable support = disabling support + reenabled support

Reply:
So that means this loop will be continuously running as long as the addon is disabled in the session? 🤔

Reply:
Now that you say that, this should probably stop and restart with disable/enable, I'll do that later

Reply:
Would this be better?

Reply:
It's still not ideal, as you could still have two active while loops at the same time.

Reply:
That shouldn't have too much impact, unless you'd disable/enable the addon a lot of times.

Reply:
It can happen by reenabling just once

Reply:
Yes, but that shouldn't be too bad. There's not really a way to prevent it.

Reply:
Probably shouldn't be too bad, but there's always a way.